pom.xml

```diff
@@ -904,6 +914,7 @@
         <module>sdk/eventhubs</module>
         <module>sdk/extendedlocation</module>
         <module>sdk/formrecognizer</module>
+        <module>sdk/personalizer</module>
```

Comment:
Please add this module entry in alphabetical order, it should be after the sdk/peering entry

Reply:
Fixed.